packages/enzyme-test-suite/test/selector-spec.jsx

```diff
@@ -294,37 +294,16 @@ describe('selectors', () => {
 
           const inputNode = wrapper.find('input');
           expect(inputNode.is(':focus')).to.equal(true);
-
-        });
-
-        it('focus pseudo works on functional component in mount', () => {
```

Comment:
I didn't know that stateless component not work in react@0.13 so removed it here and rewrite other tests with class component